spec/controllers/idv/in_person/usps_locations_controller_spec.rb

```diff
@@ -121,6 +122,26 @@
         end
       end
 
+      context 'no addresses found by usps' do
+        before do
+          allow(proofer).to receive(:request_facilities).with(address).and_return(empty_locations)
```

Comment:
This is nit picky, but it seems like the pattern between this test and the `address_controller_spec.rb` are different, but essentially doing the same thing. Is there a reason why we're not doing something simple like```let(:addresses) do [] end``` in `address_search_controller.spec.rb`?

Reply:
i thought it might be a bit easier to read the test like this

Reply:
Totally agree. I was more just wondering if there was any value in remaining consistent?